include/edm4hep/Constants.h

```diff
@@ -21,12 +21,33 @@
 
 #include <cstdint>
 
+#define DEPRECATED_LABEL(name, newname)                                                                                \
+  static constexpr const auto name [[deprecated("Use 'edm4hep::labels::" #newname "' instead")]] = labels::newname
+
 namespace edm4hep {
-static constexpr const char* CellIDEncoding = "CellIDEncoding";
-static constexpr const char* EventHeaderName = "EventHeader";
-static constexpr const char* EventWeights = "EventWeightNames";
-static constexpr const char* shapeParameterNames = "shapeParameterNames";
-static constexpr const char* EventFilterStats = "EventFilterStats";
+namespace labels {
+  static constexpr const char* CellIDEncoding = "CellIDEncoding";
+  static constexpr const char* EventHeader = "EventHeader";
+  static constexpr const char* EventWeightsNames = "EventWeightNames";
+  static constexpr const char* ShapeParameterNames = "shapeParameterNames";
+  static constexpr const char* EventFilterStats = "EventFilterStats";
+
+  /// The collection parameter name for accessing the names of the parameters for
+  /// a ParticleID collection
+  static constexpr const char* PIDParameterNames = "ParameterNames";
+  static constexpr const char* PIDAlgoName = "AlgoName";
+  static constexpr const char* PIDAlgoType = "AlgoType";
+} // namespace labels
+
+DEPRECATED_LABEL(CellIDEncoding, CellIDEncoding);
+DEPRECATED_LABEL(EventHeaderName, EventHeader);
+DEPRECATED_LABEL(EventWeights, EventWeights);
```

Comment:
```suggestion
DEPRECATED_LABEL(EventWeights, EventWeightsNames);
```
Necessary fix after renaming the new label.